Global/Images.gitignore

```diff
@@ -61,3 +62,9 @@
 # 3D Images
 *.3dm
 *.max
+
+# High Efficiency Image File Format
+*.heif
+
+# Adobe PhotoShop Document
+*.psd
```

Comment:
Open Q: Is this the best place to exclude this? I'd consider this to be a "source" file for an image, and I notice that .ai files etc. aren't currently ignored here.